tests/integration/test_rest.py

```diff
@@ -199,7 +199,7 @@ async def test_topics(rest_async_client, admin_client):
     assert data["partitions"][0]["replicas"][0]["in_sync"], "Replica should be in sync"
     res = await rest_async_client.get(f"/topics/{topic_foo}")
     assert res.status_code == 404, f"Topic {topic_foo} should not exist, status_code={res.status_code}"
-    assert res.json()["error_code"] == 40401, "Error code does not match"
+    assert res.json()["error_code"] == 40403, "Error code does not match"
```

Comment:
I didn't want to update the whole `test_rest.py` right now, there is lots of places using the hardcoded constant and not the enum there.